Development workflow python build fails : No rule to make target _ccllib

I'm following the Development Workflow instructions. I get no errors during C compilation. However the python build fails:

```
$ python setup.py build
running build
running build_py
CMake Deprecation Warning at /usr/share/cmake-3.12/Modules/UseSWIG.cmake:733 (message):
  SWIG_LINK_LIBRARIES is deprecated.  Use TARGET_LINK_LIBRARIES instead.
Call Stack (most recent call first):
  pyccl/CMakeLists.txt:30 (swig_link_libraries)


CMake Deprecation Warning at /usr/share/cmake-3.12/Modules/UseSWIG.cmake:733 (message):
  SWIG_LINK_LIBRARIES is deprecated.  Use TARGET_LINK_LIBRARIES instead.
Call Stack (most recent call first):
  pyccl/CMakeLists.txt:36 (swig_link_libraries)


-- Configuring done
-- Generating done
-- Build files have been written to: /home/msyriac/repos/CCL/build
make: Entering directory '/home/msyriac/repos/CCL/build'
make: *** No rule to make target '_ccllib'.  Stop.
make: Leaving directory '/home/msyriac/repos/CCL/build'
Traceback (most recent call last):
  File "setup.py", line 65, in <module>
    'Topic :: Scientific/Engineering :: Physics'
  File "/usr/lib/python3.7/site-packages/setuptools/__init__.py", line 140, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.7/distutils/command/build.py", line 135, in run
    self.run_command(cmd_name)
  File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "setup.py", line 17, in run
    raise Exception("Could not build CCL")
Exception: Could not build CCL
```

I've reproduced this in a python2 environment as well. Also note the swig-related deprecation warnings. I've checked this is unrelated to the above failure.

Sure, here's the output after starting from scratch:

```
$ python setup.py build
running build
running build_py
-- The C compiler identification is GNU 8.2.1
-- The CXX compiler identification is GNU 8.2.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.5.3") 
-- Found FFTW: /usr/include  
-- Found GSL: /usr/include (found suitable version "2.5", minimum required is "2.1") 
-- Found SWIG: /usr/bin/swig (found version "3.0.12") 
-- Found PythonInterp: /usr/bin/python (found version "3.7") 
-- Found PythonLibs: /usr/lib/libpython3.7m.so
-- Found NumPy: version "1.15.1" /home/msyriac/.local/lib/python3.7/site-packages/numpy/core/include
CMake Deprecation Warning at /usr/share/cmake-3.12/Modules/UseSWIG.cmake:733 (message):
  SWIG_LINK_LIBRARIES is deprecated.  Use TARGET_LINK_LIBRARIES instead.
Call Stack (most recent call first):
  pyccl/CMakeLists.txt:30 (swig_link_libraries)


CMake Deprecation Warning at /usr/share/cmake-3.12/Modules/UseSWIG.cmake:733 (message):
  SWIG_LINK_LIBRARIES is deprecated.  Use TARGET_LINK_LIBRARIES instead.
Call Stack (most recent call first):
  pyccl/CMakeLists.txt:36 (swig_link_libraries)


-- Configuring done
-- Generating done
-- Build files have been written to: /home/msyriac/repos/CCL/build
make: Entering directory '/home/msyriac/repos/CCL/build'
make: *** No rule to make target '_ccllib'.  Stop.
make: Leaving directory '/home/msyriac/repos/CCL/build'
Traceback (most recent call last):
  File "setup.py", line 65, in <module>
    'Topic :: Scientific/Engineering :: Physics'
  File "/usr/lib/python3.7/site-packages/setuptools/__init__.py", line 140, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.7/distutils/command/build.py", line 135, in run
    self.run_command(cmd_name)
  File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "setup.py", line 17, in run
    raise Exception("Could not build CCL")
Exception: Could not build CCL
```

--- a/pyccl/CMakeLists.txt
+++ b/pyccl/CMakeLists.txt
@@ -3,7 +3,7 @@
 #
 include(BuildSWIG)
 include(UseSWIG)
-set (UseSWIG_TARGET_NAME_PREFERENCE STANDARD)
+#set (UseSWIG_TARGET_NAME_PREFERENCE STANDARD)
 
 find_package(PythonLibsNew)
 find_package(NumPy)

